src/Microsoft.HttpRepl/DirectoryStructureExtensions.cs

```diff
@@ -15,12 +16,16 @@ public static IEnumerable<string> GetDirectoryListingAtPath(this IDirectoryStruc
 
         public static IDirectoryStructure TraverseTo(this IDirectoryStructure structure, string path)
         {
+            path = path ?? throw new ArgumentNullException(nameof(path));
+
             string[] parts = path.Replace('\\', '/').Split('/');
             return structure.TraverseTo(parts);
```

Comment:
do you need to check `structure` for null too?  It's checked in the next overload (and that one doesn't check `pathParts` for null...)

Reply:
Yep. Done.